pkg/cluster/member.go

```diff
@@ -34,7 +34,7 @@ func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 		if c.cluster.Spec.SelfHosted != nil {
 			name = m.Name
 			if len(name) == 0 || len(m.ClientURLs) == 0 {
-				c.logger.Errorf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
+				c.logger.Warningf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
```

Comment:
Well. It's quite relevant since we warn it in the self-hosted loop. And it is good to fix it one way or another.